.env.example

```
@@ -183,6 +183,7 @@ SEARXNG_HOSTNAME=searxng.yourdomain.com
 SUPABASE_HOSTNAME=supabase.yourdomain.com
 WAHA_HOSTNAME=waha.yourdomain.com
 WEAVIATE_HOSTNAME=weaviate.yourdomain.com
+UPTIME_KUMA_HOSTNME=uptime-kuma.yourdomain.com
```

dotenv-linter (4.0.0)

[warning] [UnorderedKey] The UPTIME_KUMA_HOSTNME key should go before the USER_DOMAIN_NAME key

(UnorderedKey)